ex2.yml

```diff
@@ -4,8 +4,7 @@ alpha: 0.125
 global_size: { height: 60, width: 12 }
 # degree of parallelism (number of blocks in each dimension)
 parallelism: { height: 1, width: 1 }
-# PDI configuration
+
 pdi:
-  plugins:
-    trace:
-      logging: { pattern: '[PDI][%n-plugin] *** %l: %v' }
+#*** PDI configuration: use the trace plugin to print all the PDI activities
```

Comment:
do you want people to use `trace logging` or just `trace`?
looking at the .log file, I suppose only `trace`. Right?

Reply:
The plugin trace give the time when you use pdi_share, ... Hence, to compare the logs, we need to remove the time. For this reason, we add the line with "logging" keyword

Reply:
OK, I see the line in the readme explain `logging`. great